imports/api/api.js

```diff
@@ -0,0 +1,6 @@
+import './github/methods'
+import './pages/methods'
+import './projects/methods'
+
+import './pages/publications'
+import './projects/publications'
```

Comment:
It's just running the code in those files, which registers the methods and publications.  There's nothing to export.